### ISSUE TITLE
Add env example with list of all possible env variables

### ISSUE THREAD

**Issue by luvnft** [imputnet/cobalt#535]
Can you update a ENV key example?

**Comment by dumbmoron** [comment]
> Can you update a ENV key example?

what?

**Comment by lostdusty** [comment]
i think op wants examples of the env variables used

**Comment by wukko** [comment]
you can find the list of all environment variables here: https://github.com/imputnet/cobalt/blob/current/docs/run-an-instance.md#list-of-all-environment-variables

**Comment by wukko** [comment]
could you rephrase your question? not really sure what you mean by "can you update env example"

**Comment by GreeningSiren** [comment]
> could you rephrase your question? not really sure what you mean by "can you update env example"

Maybe he wants a .env.example file 

env.example is the file that has every environmental value that .env has but with no value

**Comment by wukko** [comment]
as mentioned earlier, we have tables of all possible ENV variables in api & web readme files.
